guides/release/templates/actions.md

```diff
@@ -20,9 +20,10 @@ export default class Post extends Component {
   }
 }
 ```
+`@action` lets `toggleBody()` access the component's other properties when invoked from the template with `this`. Without the decorator, `this.isShowingBody` in `toggleBody()` is undefined. For more information on `this` [Understanding Javascript Function Invocation and this](https://yehudakatz.com/2011/08/11/understanding-javascript-function-invocation-and-this/)
```

Comment:
Thanks! My one suggestion is that the first sentence suggests that we use `this` to invoke the method. Perhaps this is syntactically clearer:

>`@action` lets `toggleBody()` access the component's other properties with `this` when it is invoked from the template.

Reply:
I think we should be clear that `@action` binds `this` to the instance of the class. And in turn, you can access component properties because of that.

Reply:
I like the use of the technical description (the keyword "binds") along with the lay description of what the result is.

Reply:
Adjusted! :)